src/elements/SingleReview.jsx

```diff
@@ -42,41 +53,79 @@ export const SingleReview = ({
       });
   };
 
+  if (!reviewLoaded) return "Loading Review ...";
+  if (!review) return <span className="error">404: couldn't find review</span>;
   return (
     <div className="SingleReview">
       <div className="spreadLine">
-        <h2>{title} </h2>
-        <div className="noWrap">
+        <h2>{review.title} </h2>
+        <span className="noWrap">
           {localVotes > 1 ? `${localVotes} votes` : `${localVotes}vote`}{" "}
           <button
             className="voteButton"
             disabled={processingVote}
             onClick={vote}
           >
-            {localVotes > votes ? "▼undo" : "▲vote"}
+            {localVotes > review.votes ? "undo▼" : "vote▲"}
           </button>
-        </div>
+        </span>
       </div>
-
       <p>
-        Written by {owner}, {moment(created_at).format("MMM Do YYYY")},
+        Written by {review.owner},{" "}
+        {moment(review.created_at).format("MMM Do YYYY")},
       </p>
 
-      <img src={review_img_url} alt="header image of game"></img>
+      <img src={review.review_img_url} alt="header image of game"></img>
       <p>
-        A {category} game by {designer}
+        A {review.category} game by {review.designer}
       </p>
+      <p>{review.review_body}</p>
 
-      <p>{review_body}</p>
+      {comments.length === 0 ? null : (
+        <h3>
+          {comments.length} {comments.length !== 1 ? "Comments" : "Comment"}:
+        </h3>
+      )}
 
-      <h3>
-        {comment_count} {comment_count !== 1 ? "Comments" : "Comment"}:{" "}
-      </h3>
       {commentsLoaded ? (
         <ul>
           {comments.map((comment) => (
             <CommentCard key={comment.comment_id} comment={comment} />
           ))}
+          <h3>Comment:</h3>
+          <form>
+            <textarea
+              name="comments"
+              id="comments"
+              placeholder="Write your comment ..."
+              value={newCommentText}
+              onChange={(e) => {
+                setNewCommentText(e.target.value);
+              }}
+            ></textarea>
```

Comment:
Currently I can post a blank comment, a user shouldn't be able to post a empty comment